src/helper.cc

```diff
@@ -312,6 +316,13 @@ helperOpenServers(const helper::Pointer &hlp)
         comm_read(srv->readPipe, srv->rbuf, srv->rbuf_sz - 1, call);
     }
 
+    // Call handleLackOfServers() before hlp->last_restart is updated because
+    // that method uses last_restart to measure the delay since previous start.
+    // TODO: Refactor last_restart code to measure failure frequency rather than
+    // detecting a helper X failure that is being close to the helper Y start.
```

Comment:
Let's clarify this to avoid implying that this is about two different helpers. It is about two different helper processes:

```suggestion
    // detecting a helper #X failure that is being close to the helper #Y start.
```

Reply:
```C++
InstanceIdDefinitions(HelperServerBase, "Hlpr");
```

In this context, it kind of implies that IMO because that is how we (sometimes) report individual helper processes -- Hlpr#4. There is certainly no consistency or established mnemonic here. We can make this TODO comment a lot more clear/explicit, of course, but it would take more words, and, hence, is more likely to trigger even more change requests.

If you prefer, please feel free to replace "helper" with "process" or "helper process".